syntax = "proto3";

message FanPumpStatusConfig{

    string  Status = 1;  
	string  WorkMode = 2; 

}